### PR TITLE
member entity created

### PR DESCRIPTION
**Pull request by DenisFigula**

### CODE REVIEW

**Comment by DenisFigula** [comment]
review please :)

File: pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Gender.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package cz.muni.fi.pa165.photographyclub;
+
+public enum Gender {
+    MALE, FEMALE, APACHE
```
Comment (melanger) [inline pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Gender.java]: Should we dig into something more than male and female?

File: pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Member.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    private String motivation;
+
+    @Column
+    private String experience;
+
+    @Column
+    private Gender gender;
+
+    @Column
+    private BufferedImage photo;
```
Comment (melanger) [inline pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Member.java]: I would probably stick with a String with the file name (later the URL) of the image.

**Comment by melanger** [comment]
Should we include the linking fields (e.g. Equipment.owner and/or Member.equipments)?

**Comment by DenisFigula** [comment]
List of equipment should be included, in my opinion. I will add it after equipment entity is available

**Review by melanger — CHANGES_REQUESTED** [review]

File: pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Gender.java (hunk 1 above)
Comment (DenisFigula) [reply pa165-photography-club-ejb/src/main/java/cz/muni/fi/pa165/photographyclub/Gender.java]: updated

**Review by melanger — APPROVED** [review]

**Pull request merged**